src/hyperscript/h-jsx.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as assert from 'assert'
+
+import { hJsx as h } from './h-jsx'
+import { HyperscriptChildren } from '../index'
+
+describe('h jsx support', () => {
+  it('accepts a function', () => {
+    function MyComponent() {
+      return <div />
+    }
+
+    const vNode = <MyComponent />
+    assert.strictEqual(vNode.tagName, 'div')
+  })
+
+  it('supports destructuration', () => {
```

Comment:
What does “destructuration” mean? I haven’t been able to find this word in a dictionary.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

Reply:
So even though I used Object destructuring to get to the classname, I did NOT send any props when using jsx syntax, so the jsx transform will send `null` ... this is to test that the `props === null ? {} : props` works correctly.

Reply:
cc @TylorS167